recipes/Cbc/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "coin-or-cbc" %}
+{% set version = "2.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/coin-or/Cbc/archive/releases/{{ version }}.tar.gz
+    sha256: 601a7084e75955f98ccf1ef982d272cbe7f34558c527dadc661d49e6f0fc58a3
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make  # [unix]
+  host:
+    - coin-or-utils
+    - coin-or-cgl
+    - coin-or-clp
+    - coin-or-osi
+    - libblas
+    - libcblas
+    - liblapack
+    - readline  # [unix]
+    - zlib
+    - bzip2
+    # - mumps
+    # - nauty
+    - ncurses
+    # TODO upstream should link with blas, not openblas
+    - openblas
+  run:
+    - ncurses
+    - openblas
+  run_constrained:
+    - coincbc * metapackage
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libCbc${SHLIB_EXT}
+    - test -f $PREFIX/include/coin/CbcConfig.h
+    - test -f $PREFIX/lib/libCbcSolver${SHLIB_EXT}
+    - test -f $PREFIX/lib/libOsiCbc${SHLIB_EXT}
+
+about:
+  home: https://projects.coin-or.org/Cbc
+  dev_url: https://github.com/coin-or/Cbc
+  license: EPL-2.0
+  license_family: OTHER
+  license_file: LICENSE
+  summary: 'COIN-OR branch and cut (Cbc)'
+  description: |
+    Cbc (Coin-or branch and cut) is an open-source mixed integer linear
+    programming solver written in C++. It can be used as a callable library or
+    using a stand-alone executable. It can be called through AIMMS (through the
+    AIMMSlinks project), AMPL (natively), CMPL, GAMS (through the GAMSlinks
+    project), JuMP, Mathematica, MiniZinc, MPL (through the CoinMP project), PuLP,
+    Python (e.g., cbcpy), and OpenSolver for Excel, among others.
+
+extra:
+  recipe-maintainers:
+    - wolfv
+    - tkralphs
```

Comment:
@tkralphs - are you OK with being a maintainer?

Reply:
Yeah, I'm OK.